src/main/java/com/minerarcana/floralchemy/FloraEventHandler.java

```diff
@@ -0,0 +1,16 @@
+package com.minerarcana.floralchemy;
+
+import net.minecraft.world.WorldServer;
+import net.minecraftforge.event.world.WorldEvent;
+import net.minecraftforge.fml.common.Mod.EventBusSubscriber;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@EventBusSubscriber(modid = Floralchemy.MOD_ID)
+public class FloraEventHandler {
+    @SubscribeEvent
+    public static void onWorldLoad(WorldEvent.Load event) {
+        if(!event.getWorld().isRemote) {
+            event.getWorld().addEventListener(new WorldEventListener((WorldServer) event.getWorld()));
```

Comment:
You need to protect against the sillies like me when I screwed up inMoarcarts and passed World everywhere.